### PR TITLE
working on other enity like cherry

### PR DESCRIPTION
**Pull request by vinaymodi-galaxy**
-implement canvas on scene
-implement cherry animation 
-create score board and display score on it
-code clean up and commenting the code
-set dynamically jump and speed field from unity engine

### CODE REVIEW

File: Assets/Forg.cs [hunk 1]
```diff
@@ -0,0 +1,88 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Forg : MonoBehaviour
```
Comment (sanjaykushwah06) [inline Assets/Forg.cs]: Could you please add comment what this class does

File: Assets/Script/PlayerController.cs [hunk 2]
```diff
@@ -0,0 +1,161 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+
+public class PlayerController : MonoBehaviour
```
Comment (sanjaykushwah06) [inline Assets/Script/PlayerController.cs:7]: same here

**Review by sanjaykushwah06 — CHANGES_REQUESTED** [review]

File: Assets/Script/PlayerController.cs (hunk 2 above)
Comment (vinaymodi-galaxy) [reply Assets/Script/PlayerController.cs:7]: Done

File: Assets/Forg.cs (hunk 1 above)
Comment (vinaymodi-galaxy) [reply Assets/Forg.cs]: Done

**Review by sanjaykushwah06 — APPROVED** [review]

**Pull request merged**